docs/index.html

```diff
@@ -19,6 +19,7 @@
             <h3 class="float-md-start mb-0">Jambot</h3>
             <nav class="nav nav-masthead justify-content-center float-md-end">
                 <a class="nav-link fw-bold py-1 px-0 active" aria-current="page" href="#">Home</a>
+                <a class="nav-link fw-bold py-1 px-0" href="/getting-started.html">Development</a>
```

Comment:
Inconsistency, on the getting-started.html it says 'Getting started' and here 'Development'